src/Winton.Extensions.Configuration.Consul/Winton.Extensions.Configuration.Consul.csproj

```diff
@@ -16,7 +16,7 @@
     <PackageVersion>$(NuGetVersion)</PackageVersion>
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>https://github.com/wintoncode/Winton.Extensions.Configuration.Consul</RepositoryUrl>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
We need to maintain `net461` as a TFM because of the conditional compilation below for `System.Net.Http` etc otherwise there are runtime errors when using this lib on `net461` due to the way Consul has referenced it's deps on that TFM.